extras/sdio/sdio.c

```diff
@@ -141,6 +141,8 @@ static uint16_t crc_ccitt(const uint8_t *data, size_t n)
 #define spi_skip_word()   do { spi_read_byte(); spi_read_byte(); } while(0)
 #define spi_skip_dword()  do { spi_read_byte(); spi_read_byte(); spi_read_byte(); spi_read_byte(); } while(0)
 
+#define timeout_expired(start, len) ((uint32_t)(sdk_system_get_time() - (start)) >= (len))
```

Comment:
Isn't it still possible for this to be problematic if sdk_system_get_time wraps before len elapsed, meaning start > sdk_system_get_time?